app/models/user.rb

```diff
@@ -0,0 +1,42 @@
+class User < ApplicationRecord
+  has_many :orders
+  attr_accessor :remember_token
+
+  validates :name, presence: true, length: { maximum: 50 }
+  before_save { email.downcase! }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
I like it that you also added a format validation, nice!

Rails also a regex for that, so you could use: `format: { with: URI::MailTo::EMAIL_REGEXP }`, but your implementation also works